concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3419,6 +3419,30 @@ jobs:
           - get: cf-acceptance-tests
 
       - do:
+        - task: ensure-custom-error-page-is-presented
+          tags: [colocated-with-web]
+          config:
+            platform: linux
+            image_resource: *cf-cli-image-resource
+            params:
+              SYSTEM_DNS_ZONE_NAME: ((system_dns_zone_name))
+            run:
+              path: bash
+              args:
+                - -c
+                - |
+                  set -ueo pipefail
+
+                  TEST_PAGE="https://404.${SYSTEM_DNS_ZONE_NAME}"
+
+                  echo "Hitting ${TEST_PAGE}"
+                  HTML_CONTENT="$(curl "${TEST_PAGE}")"
+
+                  echo "Testing for HTML content..."
+                  grep 'class="govuk-header"' <(echo "${HTML_CONTENT}") > /dev/null
+                  grep 'Not Found' <(echo "${HTML_CONTENT}") > /dev/null
+                  grep 'Requested route (&#39;404.tlwr.dev.cloudpipeline.digital&#39;) does not exist.' <(echo "${HTML_CONTENT}") > /dev/null
```

Comment:
This needs to be `((apps_domain))` or something

Reply:
addressed